### PR TITLE
[SD-3627] Content Expiry - Separating Archived Collection to another database 

### PR DESCRIPTION
**Pull request by mdhaman**

### CODE REVIEW

**Comment by sivakuna-aap** [comment]
Shouldn't the settings.py have the `ARCHIVED_DBNAME` and `ARCHIVED_URI` similar to `LEGAL_ARCHIVE_URI`?

**Comment by mdhaman** [comment]
@sivakuna-aap this is core PR.

Moreover,  there are no references to `LEGAL_ARCHIVE_URI` in the https://github.com/superdesk/superdesk-core/blob/master/superdesk/factory/settings.py . Those settings will be in the `settings.py` of superdesk repo.

 imo, there should not be any references to `ARCHIVED` and `LEGAL` database in the core. This could be refactored but not as part of this PR.

**Comment by sivakuna-aap** [comment]
ok. :+1:

File: superdesk/factory/settings.py [hunk 1]
```diff
@@ -189,21 +189,14 @@ def env(variable, fallback_value=None):
 # The number of minutes since the last update of the Mongo auth object after which it will be deleted
 SESSION_EXPIRY_MINUTES = 240
 
-# The number of minutes before spiked items purged
-SPIKE_EXPIRY_MINUTES = 300
-
 # The number of minutes before content items purged
 # akin.tolga 06/01/2014: using a large value (30 days) for the time being
-CONTENT_EXPIRY_MINUTES = 43200
+CONTENT_EXPIRY_MINUTES = 4320
```
Comment (akintolga) [inline superdesk/factory/settings.py]: Minor issue: comment here needs to change...

**Comment by akintolga** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**